lib/main.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'movie.dart';
+import 'movie_main_widget.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+  static const double appBarElevation = 0;
+
+
+  // This widget is the root of your application.
```

Comment:
delete this extra line and unnecessary comment

Reply:
done